compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/bala/CompilerPluginJson.java

```diff
@@ -0,0 +1,26 @@
+package io.ballerina.projects.internal.bala;
```

Comment:
missing license header

Reply:
Added in fadd4484f31c0081894485782cb944264a6f84e2